admin.adoc

```diff
@@ -547,7 +547,7 @@ You'll need a copy of the GRR source:
 > git clone https://github.com/google/grr.git
 --------------------------------------------------------------------------------
 
-and the latest versions of link:https://www.vagrantup.com/[Vagrant] and VirtualBox installed.
+and the latest versions of link:https://www.vagrantup.com/[Vagrant] and VirtualBox installed. The Linux VM's provided by the project were built with an older version of VirtualBox. It is recommended that you install the plugin link:https://github.com/dotless-de/vagrant-vbguest[vagrant-vbguest] to update the guest additions to the current version your running on the build host.
 
```

Comment:
So yeah, I saw this problem after a reboot, new kernel. I'm going to merge this then change it a bit.